### PR TITLE
Update env in yml files

### PR DESCRIPTION
**Pull request by habib-deriv**
Update env in yml files

### CODE REVIEW

File: src/main.tsx [hunk 1]
```diff
@@ -30,6 +30,8 @@ const AnalyticsConfigurator = () => {
         if (websiteStatusData?.clients_country) {
             const accountType = activeTradingAccount?.is_virtual ? 'demo' : 'real';
             const clientCountry = websiteStatusData.clients_country;
+            // eslint-disable-next-line no-console
+            console.log('RudderStack Key:', import.meta.env.VITE_RUDDERSTACK_KEY);
```
Comment (thisyahlen-deriv) [inline src/main.tsx:34]: Do we need this ?

Comment (habib-deriv) [reply src/main.tsx:34]: Yeah kept it temporary.. will remove this soon

Comment (habib-deriv) [inline src/main.tsx:34]:
```suggestion
```

File: src/main.tsx [hunk 2]
```diff
@@ -30,6 +30,7 @@ const AnalyticsConfigurator = () => {
         if (websiteStatusData?.clients_country) {
             const accountType = activeTradingAccount?.is_virtual ? 'demo' : 'real';
             const clientCountry = websiteStatusData.clients_country;
+            // eslint-disable-next-line no-console
```
Comment (thisyahlen-deriv) [inline src/main.tsx:33]: can remove this

Comment (habib-deriv) [inline src/main.tsx:33]:
```suggestion
```

**Review by thisyahlen-deriv — APPROVED** [review]

**Pull request merged**